### DISCUSSION TITLE
Potential Cost

### DISCUSSION THREAD

**Discussion by Phil1216** [aws-samples/websocket-chat-application#25] [Q&A]
If I were to deploy this sample app and leave it around for a while so I can play with it, what sort of costs might I incur? Which parts of the deployment have the greatest potential to incur cost?

**Comment by santatamas** [comment]
Hi Phil! Thanks for reaching out!
As far as I'm aware, all infrastructure components are serverless and has pay-per-use cost model. So it should be safe to leave it running without incurring massive cost. I can provide an accurate cost estimation next week :)

**Comment by santatamas** [comment]
Hi Phil,

I have checked the infrastructure components, and if you make no requests to the APIs (so no active users), nothing should incur costs other than the S3 bucket that stores the static website. The overall cost should be pennies/month in the worst case. I wanted to give you a detailed cost estimation, but you'd have $0 for most of the items, except S3 :) I hope this helps!
